services/ui-src/src/components/cards/EntityCard/EntityCard.test.tsx

```diff
@@ -46,23 +44,10 @@ const mockUnfinishedEntityData = {
   standardType: "mock-standardType",
 };
 
-const EntityCardComponent = (
-  <EntityCard
-    entity={mockEntity}
-    entityType={mockEntityType}
-    formattedEntityData={mockFormattedEntityData}
-    verbiage={mockModalDrawerReportPageJson.verbiage}
-    openAddEditEntityModal={openAddEditEntityModal}
-    openDeleteEntityModal={openDeleteEntityModal}
-    openDrawer={mockOpenDrawer}
-    data-testid="mock-entity-card"
-  />
-);
```

Comment:
Question: Don't need to test this anymore?

Reply:
Nope any tests for this case apply to any of the actual entity-named components